src/index.ts

```diff
@@ -82,6 +82,7 @@ import {TwitterConnect} from './plugins/twitter-connect';
 import {Vibration} from './plugins/vibration';
 import {VideoPlayer} from './plugins/video-player';
 import {WebIntent} from './plugins/webintent';
+import {CodePush} from './plugins/code-push';
```

Comment:
import/export statements are arranged by alphabetical order, please move this and the rest of the import/export statement to the appropriate place